app/views/core_induction_programmes/modules/show.html.erb

```diff
@@ -5,10 +5,10 @@
       <%= govuk_link_to "Edit module content", edit_module_path(@course_module), button: true%>
     <% end %>
 
-    <% if @cip_is_ambition %>
-      <h1 class="govuk-heading-l"> <%= "#{@course_module.term.capitalize} term: #{@course_module.title}" %></h1>
+    <% if @course_module.page_header? %>
```

Comment:
What happens when `page_header` is an empty string?
I expected `@course_module.page_header.present?`, but maybe that is enough :)

Reply:
Yeah looks to be enough. Adding an empty string doesn't change the title , it goes to term_and_title